src/discussions/posts/post-editor/PostEditor.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useDispatch } from 'react-redux';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button, Form } from '@edx/paragon';
+import { Help, Post } from '@edx/paragon/icons';
```

Comment:
I don't know how you are using these icons with 14.5.0 version of paragon these icons are not available in that version.

Reply:
Yes, I think I forgot to include the updated package.* files. Will update.